fedbiomed/common/secagg/_dh.py

```diff
@@ -0,0 +1,122 @@
+from Crypto.PublicKey import ECC
+from Crypto.Hash import TupleHash128
+from Crypto.Protocol.DH import key_agreement
+from fedbiomed.common.exceptions import FedbiomedSecaggCrypterError
+from fedbiomed.common.constants import ErrorNumbers
+
+
+class DHKeysGeneration:
```

Comment:
Hi @rtaiello, 

This class can be responsible also importing keys. For example, 

- it can take public and private key as input. Then, it imports the keys using method `_import_key_pem`. 
- if there is no public or private key provided as arguments (`None`) it generates a new one. 

Basically this class will be responsible for wrapping keys that are going to be used for DH.  If you apply this suggestion you I think you can rename this class as `DHKey`.

Reply:
Hey @rtaiello I just pushed a new commit, following you suggestions, many thx.